<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_DATA_TRANSMIT</class>
 <widget class="QDialog" name="GUI_DATA_TRANSMIT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>570</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="SaveAs_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearReceived_Button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="recv_PlainText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendMSG_Button">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>279</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="TX_Radio">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>35</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TX</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">TX_RX_Sel</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="RX_Radio">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>36</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">TX_RX_Sel</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="TX_RX_Radio">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>52</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TX/RX</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">TX_RX_Sel</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="Input_Radio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>155</y>
     <width>49</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">MSG_Sel</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="File_Radio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">MSG_Sel</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="FilePathEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>131</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenFile_Button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="msg_PlainText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Receive_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Received Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GUI_DATA_TRANSMIT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Send_Label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>100</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="MSG_Sel"/>
  <buttongroup name="TX_RX_Sel"/>
 </buttongroups>
</ui>
